src/pdc-api.ts

```diff
@@ -21,11 +21,11 @@ const logError = (error: unknown, path: string, params: object) => {
 const usePdcApi = <T>(
   path: string,
   params: URLSearchParams = new URLSearchParams(),
-): T | null => {
+): [T | null, () => void] => {
   const { fetch } = useOidcFetch();
   const [response, setResponse] = useState(null);
 
-  useEffect(() => {
+  const fetchData = useCallback(async () => {
```

Comment:
This doesn't need to be async / it's not using await anywhere and it returns a void.